.github/workflows/cloud-regress.yml

```diff
@@ -73,15 +79,42 @@ jobs:
           path: /tmp/neon/
           prefix: latest
 
+      - name: Create a new branch
+        id: create-branch
+        uses: ./.github/actions/neon-branch-create
+        with:
+          api_key: ${{ secrets.NEON_STAGING_API_KEY }}
+          project_id: ${{ vars[format('PGREGRESS_PG{0}_PROJECT_ID', matrix.pg-version)] }}
+
+      - name: Wait for operations to be completed
+        uses: ./.github/actions/neon-wait-operations-to-finish
+        with:
+          api_key: ${{ secrets.NEON_STAGING_API_KEY }}
+          project_id: ${{ vars[format('PGREGRESS_PG{0}_PROJECT_ID', matrix.pg-version)] }}
+
+      - name: Get the connection URI
+        id: connect-uri
+        uses: ./.github/actions/neon-get-connection-uri
+        with:
+          api_key: ${{ secrets.NEON_STAGING_API_KEY }}
+          project_id: ${{ vars[format('PGREGRESS_PG{0}_PROJECT_ID', matrix.pg-version)] }}
```

Comment:
`neon-branch-create` returns new connections string, can we use it?

Reply:
Yes, we can